Ifzan/charcount.js

```diff
@@ -0,0 +1,31 @@
+function abc(q)
+{
+    let count = q.length;
+    console.log(count);
+    let m = new Map();
+    for (let i=0; i<count; i++)
+    {
+        m.set(q[i],0);
+    }
+    if(count == 0)
+    {
+        console.log("Invalid");
+        return; 
+    }
+    else
+    {
+    for (let i=0; i<count; i++)
+    {
+        let a = m.get(q[i]);
+        if(m.has(q[i]))
+           {
+            m.set(q[i],a+1);
+        }
```

Comment:
what if line no 20 returns false.. you should add it with 1 as value right?

Reply:
I still have doubts regarding map function and its usage.